celery_with_django/main_app/tasks.py

```diff
@@ -0,0 +1,12 @@
+from celery import shared_task
+from time import sleep
+
+@shared_task(bind=True)
+def add(self,x,y):
```

Comment:
Can you add `x:int, y:int` type definitions here?

Reply:
yes, I added.